### PR TITLE
Refs [TASK] [35349] Design UI

### PR DESCRIPTION
**Pull request by khoaqh-2470**
## Related Tickets
- [#Task 35349: Thiết kế giao diện](https://edu-redmine.sun-asterisk.vn/issues/35349)
## WHAT
- 
Screenshot from 2021-04-16 13-48-42

## Review Checklist

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://github.com/framgia/coding-standards/tree/master/eng/android |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Conventions | Kotlin coding conventions | https://kotlinlang.org/docs/coding-conventions.html |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Sun* Redmine working process  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md |<li>- [ ] yes</li>|<li>- [ ] yes</li>  

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/java/com/sun/moviedb_53/data/source/remote/MovieRemoteDataSource.kt [hunk 1]
```diff
@@ -77,10 +75,32 @@ class MovieRemoteDataSource : MovieDataSource.Remote {
...
+    override fun <T> getDataInActor(
+        idActor: Int,
+        actorDetailType: ActorDetailType,
+        listener: OnFetchDataJsonListener<T>
+    ) {
+        val stringUrl = Constant.BASE_URL +
+                ACTOR_TYPE + idActor +
+                actorDetailType.path +
+                endPointParams
+        when (actorDetailType) {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/moviedb_53/data/source/remote/MovieRemoteDataSource.kt]:
```suggestion
                endPointParams
                
        when (actorDetailType) {
```

File: app/src/main/java/com/sun/moviedb_53/data/source/remote/MovieRemoteDataSource.kt [hunk 2]
```diff
@@ -77,10 +75,32 @@ class MovieRemoteDataSource : MovieDataSource.Remote {
...
     }
 
+    override fun <T> getDataInActor(
+        idActor: Int,
+        actorDetailType: ActorDetailType,
+        listener: OnFetchDataJsonListener<T>
+    ) {
+        val stringUrl = Constant.BASE_URL +
+                ACTOR_TYPE + idActor +
+                actorDetailType.path +
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun/moviedb_53/data/source/remote/MovieRemoteDataSource.kt]:
```suggestion
        val stringUrl = Constant.BASE_URL +
                ACTOR_TYPE + 
                idActor +
                actorDetailType.path +
```

File: app/src/main/res/layout/fragment_actor.xml [hunk 3]
```diff
@@ -131,6 +131,18 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/textAddress" />
 
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/textBiography"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/dp_10"
+        android:layout_marginLeft="@dimen/dp_10"
```
Comment (ghost) [inline app/src/main/res/layout/fragment_actor.xml:139]: android:layout_marginStart="@dimen/dp_10"
Báo warning hả hay sao mà phải để thêm : android:layout_marginLeft="@dimen/dp_10" 
vậy

**Review by tuanlvt-1117 — APPROVED** [review]

File: app/src/main/res/layout/fragment_actor.xml (hunk 3 above)
Comment (khoaqh-2470) [reply app/src/main/res/layout/fragment_actor.xml:139]: Xóa đi nó báo đỏ luôn á a.
Lúc nãy em kéo thả nên nó tự sinh ra á

**Pull request merged**